dragonfly/implementations/daq_run_interface.py

```diff
@@ -332,4 +338,324 @@ def save_trace(self, trace, comment):
             logger.debug("things are about to be dumped in file")
             json.dump(result_meta, outfile, indent=4)
             logger.debug("things have been dumped in file")
-        logger.info("saving {}: successful".format(path))
+            logger.info("saving {}: successful".format(path))
+
+
+
+__all__.append('ROACH1ChAcquisitionInterface')
+class ROACH1ChAcquisitionInterface(DAQProvider):
+    '''
+    A DAQProvider for interacting with ROACH-Psyllid DAQ
+    '''
+    def __init__(self,
+                 psyllid_interface='psyllid_interface',
+                 daq_target = 'roach2_interface',
+                 channel = 'a',
+                 filename_prefix = 'psyllid',
```

Comment:
remove `filename_prefix`, as it is a defined variable in DAQProvider (unless you really want to overwrite the default value)